src/Main.elm

```diff
@@ -24,7 +26,7 @@ init =
 
 -- UPDATE
 
-type Msg = Increment | Decrement
+type Msg = Increment | Decrement | Reset
 
```

Comment:
What is this line saying? Is Increment / Decrement something known that is imported, or are they being defined here? I'd normally read this like "the Msg type can be an Increment or a Decrement" but I didn't see where those are defined.

Reply:
The latter. This line is defining a new 'custom type' in Elm lingo. Think of it like an enum but even cooler. 

My PR extended the custom type by adding a third option - 'Reset'